.github/workflows/continuous-integration.yml

```diff
@@ -4,7 +4,7 @@
 name: CI Workflow
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
This keeps internal contributions as is right?

Reply:
Yeah, it still triggers for internal contributions, as well as forked repos now